pkg/model/common.go

```diff
@@ -22,3 +24,8 @@ func ApplicationKindStrings() []string {
 	}
 	return out
 }
+
+// BuildGitPathID builds a unique path between repositories.
+func BuildGitPathID(repoID, path, configFileName string) string {
```

Comment:
I just want to look into if the application is registered, using GitPath. I'm still wondering if we can do that in other ways.